frontend/lib/Registeration.dart

```diff
@@ -0,0 +1,200 @@
+import 'package:flutter/material.dart';
+import 'package:image_picker/image_picker.dart';
+import 'dart:io';
+
+class RegisterationPage extends StatefulWidget {
+  @override
+  _RegisterationPageState createState() => _RegisterationPageState();
+}
+
+class _RegisterationPageState extends State<RegisterationPage> {
+  var _index = 0;
+
+  File _image;
+  imgfromGallery() async {
+    print('picker is called');
+    File image = await ImagePicker.pickImage(source: ImageSource.gallery);
```

Comment:
`ImagePicker.pickImage` is deprecated. Replace it with the non-deprecated function.